src/BaGet/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -90,11 +90,12 @@ public static IServiceCollection AddBaGetContext(this IServiceCollection service
 
             services.AddDbContext<SqlServerContext>((provider, options) =>
             {
+
                 var databaseOptions = provider.GetRequiredService<IOptions<BaGetOptions>>()
                     .Value
                     .Database;
 
-                options.UseSqlServer(databaseOptions.ConnectionString);
+                options.UseSqlServer(databaseOptions.ConnectionString, t => t.UseRowNumberForPaging());

```

Comment:
Could you say more about the motivations for this change? It seems that this is used primarily for SQL Server 2008 support, but it also seems that EF Core has had bugs around this mode. I think I would rather avoid this option unless there's a very strong reason to add it in.

Reply:
Also, please remove the extra new line on line 93.

Reply:
yes, this change is specially for mssql 2008(R2 SP3 I use), EF generate queries with 'offset ... fetch' which is not supported by 2008, this some sort of quick fix this bug. and sure, I will remove